boards/xtensa/m5stickc_plus/doc/index.rst

```diff
@@ -0,0 +1,137 @@
+.. _m5stickc_plus:
+
+M5StickC PLUS
+##############
+
+Overview
+********
+
+M5StickC PLUS, one of the core devices in M5Stacks product series, is an ESP32-based development board.
+
+M5StickC PLUS features the following integrated components:
+
+- ESP32-PICO-D4 chip (240MHz dual core, 600 DMIPS, 520KB SRAM, Wi-Fi)
+- ST7789v2, LCD TFT 1.14", 135x240 px screen
+- IMU MPU-6886
+- SPM-1423 microphone
+- RTC BM8563
+- PMU AXP192
+- 120 mAh 3,7 V battery
+
+Some of the ESP32 I/O pins are broken out to the board's pin headers for easy access.
+
+Functional Description
+**********************
+
+The following table below describes the key components, interfaces, and controls
+of the M5StickC PLUS board.
+
+.. _ST7789v2: https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/core/ST7789V.pdf
+.. _MPU-6886: https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/core/MPU-6886-000193%2Bv1.1_GHIC_en.pdf
+.. _ESP32-PICO-D4: https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/core/esp32-pico-d4_datasheet_en.pdf
+.. _SPM-1423: https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/core/SPM1423HM4H-B_datasheet_en.pdf
+
++------------------+-------------------------------------------------------------------------+
+| Key Component    | Description                                                             |
++==================+=========================================================================+
+| 32.768 kHz RTC   | External precision 32.768 kHz crystal oscillator serves as a clock with |
+|                  | low-power consumption while the chip is in Deep-sleep mode.             |
++------------------+-------------------------------------------------------------------------+
+| ESP32-PICO-D4    | This `ESP32-PICO-D4`_ module provides complete Wi-Fi and Bluetooth      |
+| module           | functionalities and integrates a 4-MB SPI flash.                        |
++------------------+-------------------------------------------------------------------------+
+| Diagnostic LED   | One user LED connected to the GPIO pin.                                 |
++------------------+-------------------------------------------------------------------------+
+| USB Port         | USB interface. Power supply for the board as well as the                |
+|                  | communication interface between a computer and the board.               |
+|                  | Contains: TypeC x 1, GROVE(I2C+I/0+UART) x 1                            |
```

Comment:
Maybe typo? `I/O` instead of `I/0` (letter 'O' instead of zero)?

Reply:
@dcpleung it was indeed a typo, it was copied from one of the board's documentation files (now I've double checked). Changes applied.